### ISSUE TITLE
Specify font size in modeline?

### ISSUE THREAD

**Issue by AmaiKinono** [seagle0128/doom-modeline#189]
I would like to use smaller font size/height in modeline. Is there anyway to do this?

**Comment by seagle0128** [comment]
Please refer to #187. You can set smaller font height for mode-line  by set-face-attribute.

**Comment by AmaiKinono** [comment]
Thanks.

> That means the height of mode-line should be the max value of doom-modeline-height and the font height.

I don't think so... I did this:

``` emacs-lisp
(set-face-attribute 'mode-line nil :height 50)
(setq doom-modeline-height 1)
```

and the modeline looks like this:

![2019-07-01-155219_836x65_scrot](https://user-images.githubusercontent.com/28714352/60419591-3d1d7680-9c18-11e9-8e8f-b83921aa5ee7.png)

It's larger than both the maximum height of the font, and `doom-modeline-height`.

**Comment by seagle0128** [comment]
Please provide more information below.

- OS, Emacs version, doom-modeline version
- `doom-modeline-height`, `(doom-modeline--font-height)`
- error logs, messages, etc.

You'd better update to the latest version and restart Emacs.

**Comment by AmaiKinono** [comment]
- OS: Manjaro 18.0.4 with kernel 4.19.49
- Emacs version: 26.2
- doom-modeline version: commit `ee4619d3670830180ffd87b1598d74312b5b9808`. I've just pulled it so it's the latest.
- `doom-modeline-height`: 1
- `(doom-modeline--font-height)`: 44
- logs / messages: no. It's just the display issue, and nothing is reported with `emacs --debug-init`.

**Comment by seagle0128** [comment]
Obviously, the font height of mode-line is 44. That's why the mode-line looks so big. The behaviors are as designed. Again, please refer to #187.

**Comment by AmaiKinono** [comment]
Thanks. Advising `doom-modeline--font-height` now works. But it looks weird to me to set the height according to `(frame-char-height)`. would using `(face-attribute 'mode-line :height)` be better?

**Comment by seagle0128** [comment]
@AmaiKinono I've already been working on it. Will commit soon. And the advice will be simple.

**Comment by AmaiKinono** [comment]
This works. But seems the face attributes must be set or it goes wrong. I tried in `emacs -q` to require `doom-modeline` and call `doom-modeline-mode`, the modeline won't show up and emacs throws an error:

```
doom-modeline-refresh-bars: Wrong type argument: number-or-marker-p, unspecified
```

and It works after setting the face attributes.

**Comment by seagle0128** [comment]
Fix in 68362f248ee386760e7e37ae3e4babb29e68aee2.
